client/landing/stepper/declarative-flow/internals/steps-repository/processing-step/index.tsx

```diff
@@ -15,11 +16,13 @@ export enum ProcessingResult {
 	FAILURE = 'failure',
 }
 
+const SiteIntent = Onboard.SiteIntent;
+
 const ProcessingStep: Step = function ( props ): ReactElement | null {
 	const { submit } = props.navigation;
 
 	const { __ } = useI18n();
-	const loadingMessages = [
+	let loadingMessages = [
```

Comment:
Can we add this to the `default` case of the `switch` defined [below](https://github.com/Automattic/wp-calypso/pull/65456/files#diff-b7ff3243075883b8ecc07a8d6f6e9c94a960ec6d179ac29676a74d6270a6fe1aR50)? 

Also, maybe we can isolate this into a function outside the component?

Reply:
Update new [hooks](https://github.com/Automattic/wp-calypso/blob/0179f39274e9efffc1a74b4089c291167f2aea72/client/landing/stepper/hooks/use-processing-loading-messages.ts#L8) for the [processing messages](https://github.com/Automattic/wp-calypso/blob/0179f39274e9efffc1a74b4089c291167f2aea72/client/landing/stepper/declarative-flow/internals/steps-repository/processing-step/index.tsx#L23)